google-cloud-platform/index.md

```diff
@@ -9,7 +9,7 @@ has-permalinks: false
 # {{ page.title }} <img src="images/GoogleCloudPlatform-logo.png" alt="logo for Google Cloud Platform">
 
 **Note**: There are many ways to host a Dart server. You might also be
-interested in [Heroku][] and [Dartvoid][].
+interested in [Heroku][] and [Sourcevoid](https://www.sourcevoid.com/).
```

Comment:
No worries, I'm not used to that link style either. It has an academic
flavor I don't usually see in markdown.

-k-

On Tue, Jun 28, 2016 at 11:40 PM, Robert Åkerblom-Andersson <
notifications@github.com> wrote:

> In google-cloud-platform/index.md
> <https://github.com/dart-lang/server/pull/6#discussion_r68892902>:
>
> > @@ -9,7 +9,7 @@ has-permalinks: false
> >  # {{ page.title }} <img src="images/GoogleCloudPlatform-logo.png" alt="logo for Google Cloud Platform">
> >
> >  **Note**: There are many ways to host a Dart server. You might also be
> > -interested in [Heroku][] and [Dartvoid][].
> > +interested in [Heroku][] and [Sourcevoid](https://www.sourcevoid.com/).
>
> I'm sorry for the confusion on my part, I'm not used to the "[name][]"
> style links, I didn't know about that syntax variation. Now the links use
> the same style as before but with the updated name and url.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/dart-lang/server/pull/6/files/5fbaae1252d32381a50315a2714982fb9202bf39#r68892902>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACEHA6uPFqoovZKnPggWQ03CxCEYuNLrks5qQhNagaJpZM4JAEKb>
> .
>